src/Components/Auth/Login.tsx

```diff
@@ -86,7 +91,7 @@ export const Login = (props: { forgot?: boolean }) => {
   // set loading to false when component is dismounted
   useEffect(() => {
     return () => {
-      setLoading(false);
+      setLoading(queryLoading);
```

Comment:
This useEffect returns a cleanup only. You need not modify this.

```suggestion
      setLoading(false);
```